### PR TITLE
web: fork SidebarItemView into OverviewItemView

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu,

Please review the following commits I made in branch nicks/ch10493:

8490b27a2db3c2a5b293ee32b5ca47bec76135f0 (2020-12-15 12:37:16 -0500)
web: fork SidebarItemView into OverviewItemView

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
i spent some time trying to make them share a single component, but it ended up too complex (and i suspect the components might end up different enough that it's not worth it)

**Comment by nicks** [comment]
Screenshot from 2020-12-15 12-44-30

File: web/src/OverviewItemView.tsx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+          <SidebarIcon
+            tooltipText={runtimeTooltipText(item.runtimeStatus)}
+            status={item.runtimeStatus}
+            alertCount={item.runtimeAlertCount}
+          />
+          <OverviewItemName name={item.name} />
+          <OverviewItemTimeAgo>
+            {hasSuccessfullyDeployed ? timeAgo : "—"}
+          </OverviewItemTimeAgo>
+          <SidebarTriggerButton
```
Comment (hyu) [inline web/src/OverviewItemView.tsx:281]: The Trigger button shouldn't propagate clicks to its parents. Not blocking of this PR, but we should address it at some point.

**Review by hyu — APPROVED** [review]

File: web/src/OverviewItemView.tsx (hunk 1 above)
Comment (nicks) [reply web/src/OverviewItemView.tsx:281]: yep, there's a PR out for that! https://github.com/tilt-dev/tilt/pull/4003

**Pull request merged**